### PR TITLE
House Robber

### PR DESCRIPTION
**Pull request by salonishah01**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

C code for house robber problem

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#1312

### CODE REVIEW

**Comment by salonishah01** [comment]
@HarshCasper @iamrajiv Please review this PR

File: C/dp/houseRobber.c [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    return 0;
+}
+
+/*
+
+Sample input: 
+Enter number of houses: 4
+Enter money in each house: 1 2 3 1 
+Total amount that can be robbed: 4
+*/
```
Comment (vatsalkesarwani12) [inline C/dp/houseRobber.c]: add time and space complexity

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]

File: C/dp/houseRobber.c (hunk 1 above)
Comment (salonishah01) [reply C/dp/houseRobber.c]: @plazzy99 Added time and space complexity

**Comment by HarshCasper** [comment]
LGTM!

**Pull request merged**